src/services/codefixes/helpers.ts

```diff
@@ -222,7 +223,7 @@ export function addNewNodeForMemberSymbol(
         case SyntaxKind.PropertyDeclaration:
             let flags = NodeBuilderFlags.NoTruncation;

```

Comment:
This doesn't need `AllowUnresolvedNames` like the code below?

Sort of wondering if all old users of `NoTruncation` need that flag, but maybe we shouldn't add ones where we don't think it should matter except those where we would have set it manually...

Reply:
Currently is not needed to pass the test cases. Thought we might be missing some, because adding it doesn't change any of them.

Reply:
Right, our test suite is just not always exhaustive, so I suspect that there are some lurking bugs without this flag for these auto-import / codefix cases, given those seem to in other cases depend on this flag